x11-packages/qt5-qtkeychain/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/frankosterfeld/qtkeychain
```

Comment:
This package seems to be interchangeably packaged as either:
- `qt5-qtkeychain`
- `qtkeychain` or
- `qtkeychain-qt5`
https://repology.org/project/qtkeychain/information

Since we do not currently have any Qt6 packages, please consider just naming the package `qtkeychain`.